apps/pcq/pcq-backend/prod.yaml

```diff
@@ -11,8 +11,7 @@ spec:
       autoscaling:
         maxReplicas: 3
       environment:
-        NUMBER_OF_DAYS_LIMIT: 482
-        NUMBER_OF_DAYS_LESS_THAN_LIMIT: 360
+        NUMBER_OF_DAYS_LIMIT: 90
```

Comment:
The default value of this var is 90 days already pcq-backend, hence not needed in flux

Reply:
removed this just kept it like that as other environment has same